docs/guides/Integration-Guides/payments-integration-guide/google-pay-processing-information-for-payment-providers-and-anti-fraud.md

```diff
@@ -5,9 +5,40 @@ hidden: false
 createdAt: "2022-06-22T00:00:00.000Z"
 updatedAt: ""
 ---
+Here are some answers to questions you may have about transactions with Google Pay:
 
-Purchase transactions made through Google Pay are sent to payment providers and anti-fraud with the same payload structure present in a purchase made through credit or debit cards on VTEX.
+## 1. What are the differences between the payload used for a standard credit card payment and the one used for Google Pay [DPAN](https://help.vtex.com/en/tutorial/dpan-and-fpan-understanding-security-in-the-online-tokenized-payment-flow--3RM7RvhKZ057wja5xVEOqb)?
```

Comment:
```suggestion
## What are the differences between the payload used for a standard credit card payment and the one used for Google Pay [DPAN](https://help.vtex.com/en/tutorial/dpan-and-fpan-understanding-security-in-the-online-tokenized-payment-flow--3RM7RvhKZ057wja5xVEOqb)?
```

Reply:
These are not sequential steps nor have any hierarchy, I don't see why they should be numbered.